Misc/NEWS.d/next/Library/2020-10-20-13-31-32.bpo-37640.JfvXk-.rst

```diff
@@ -0,0 +1 @@
+Fix :mod:`telnetlib` crash while receiving un-printable characters in the :method:`Telnet.interact` and ``listener`` methods. 
```

Comment:
Oops, docs tests are failing since I mixed up `:meth:` and `:method:`, my bad. If the doc tests still fail, you can remove the `:meth:` role outright.
```suggestion
Fix :mod:`telnetlib` crash while receiving un-printable characters in the :meth:`Telnet.interact` and ``listener`` methods. 
```